src/Adapter/MailTemplate/MailTemplateTwigRenderer.php

```diff
@@ -135,11 +134,7 @@ private function render(
             $layoutPath = !empty($layout->getTxtPath()) ? $layout->getTxtPath() : $layout->getHtmlPath();
         }
 
-        try {
-            $renderedTemplate = $this->twig->render($layoutPath, $layoutVariables);
-        } catch (LoaderError $e) {
-            throw new FileNotFoundException(sprintf('Could not find layout file: %s', $layoutPath));
-        }
```

Comment:
It is suppressing the original error and giving false information.

If you have and error in the twig template like a `{% endblock }` is says "Could not find layout file".

Secondly, as I understand it, `FileNotFoundException` produces a nicer user message. However, that makes only sense if the user was entering the filename. I don't see how `$layoutPath` can ever be a missing file, since this was just automatically scanned before.